### PR TITLE
CFE-1128: Update base images of helm-operator to RHEL9

### PR DESCRIPTION
**Pull request by arkadeepsen**
This PR updates base images of helm-operator to RHEL9

### CODE REVIEW

File: ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.17.yaml [hunk 1]
```diff
@@ -22,11 +14,11 @@ images:
   from: root
   to: osdk-builder
 - dockerfile_path: release/helm/Dockerfile
-  from: base
+  from: base-rhel9
```
Comment (TrilokGeer) [inline ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.17.yaml:17]: Should this change be considered only when QE acknowledges all testcases are fine with rhel9 base with main branch?

Comment (arkadeepsen) [reply ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-release-4.17.yaml:17]: The tests already ran as part of rehearsals for both master and 4.17 branch. All the following tests were run and all of them passed:
- ci/rehearse/openshift/ocp-release-operator-sdk/master/e2e-helm
- ci/rehearse/openshift/ocp-release-operator-sdk/master/images 
- ci/rehearse/openshift/ocp-release-operator-sdk/master/sanity 
- ci/rehearse/openshift/ocp-release-operator-sdk/master/unit 
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/e2e-helm
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/images
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/sanity
- ci/rehearse/openshift/ocp-release-operator-sdk/release-4.17/unit

**Comment by TrilokGeer** [comment]
I'd suggest for feedback from QE execution of all testcases, if any, other than the CI e2e.

**Comment by arkadeepsen** [comment]
> I'd suggest for feedback from QE execution of all testcases, if any, other than the CI e2e.

@KeenonLee can you please provide your feedback on this?

**Comment by KeenonLee** [comment]
> > I'd suggest for feedback from QE execution of all testcases, if any, other than the CI e2e.
> 
> @KeenonLee can you please provide your feedback on this?

working on it

**Pull request merged**